src/Data/HugeNum.purs

```diff
@@ -350,12 +348,6 @@ isPositive _ = false
 isZero :: HugeNum -> Boolean
 isZero (HugeNum r) = all (_ == _zero) r.digits
 
-min :: HugeNum -> HugeNum -> HugeNum
-min x y = if x < y then x else y
-
-max :: HugeNum -> HugeNum -> HugeNum
-max x y = if x > y then x else y
```

Comment:
Removed these as `min` and `max` are provided in the `Prelude` for all `Ord`s now :)